deploy/olm-catalog/kubefed-operator/0.1.0/kubefed-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -304,75 +304,124 @@ spec:
+    **Important Note**: 
+    * This operator is purely for prototyping purposes and not
     supported by any upstream community at this time.
 
-
-    **Important Note**: Currently, while in preview, this operator does not
+    * Currently, while in preview, this operator does not
     support automatic upgrades. You must remove the old version of the operator
     manually before installing a new version.
 
 
-    ## Using KubeFed
-
-    This deploys KubeFed in a namespace-scoped as well as in a cluster-scoped fashion.
-    For example, if you subscribe to this package in namespace `my-namespace`,
-    KubeFed will be deployed to manage objects _only_ in `my-namespace` in
-    the host cluster and target clusters.
-
-    After installing KubeFed instance, you will use the `kubefedctl` command
-    line tool to do two types of configuration:
-      
-    - Configure federation to connect to your target clusters with `kubefedctl join`
-    - Create new federation APIs for the API types you want to federate with `kubefedctl enable`
-
-
-    **Important Note**: To add federation capability to your namespace, you are
-    only required to install kubefed in the hosting cluster; there is no need
-    to install kubefed in the target clusters (and no effect gained from
-    doing so).
+    ## Start using the operator
 
+    This operator deploys KubeFed in a namespace-scoped as well as in a cluster-scoped fashion. For example, if you subscribe to this package
+    in namespace `my-namespace`, KubeFed will be deployed to manage objects _only_ in `my-namespace` in the host cluster and target clusters.
+    
+    The operator provides the following CRDs for instantiating the `KubeFed` in a given cluster:
+    * KubeFedWebHook
+    * KubeFed
+    
+    ### KubeFed deployment
+    In order to deploy kubefed, you need to follow these three steps:
+
+    * Install the operator in a single `Namespace` or in All `Namespaces` based on the operator `Installation Mode`.
+    ```
+    $ kubectl get po -n <namespace>
+      NAME                               READY   STATUS    RESTARTS   AGE
+      kubefed-operator-db58d9b89-8694x   1/1     Running   0          15s
+
+    ```
+    * Create a `KubeFedWebHook` resource to instatiate an admission webhook controller for kubefed. It's recommended to install this CR with `scope:Cluster`
+    to allow webhook controller to serve API requests coming from all `kubefed` control planes within the same cluster.
+    ```
+    $ cat <<-EOF | kubectl apply -n <namespace> -f -
+    ---
+    apiVersion: operator.kubefed.io/v1alpha1
+    kind: KubeFedWebHook
+    metadata:
+      name: kubefed-webhook-resource
+    spec: 
+      scope: Cluster 
+    ---
+    ```
+    * Create a `KubeFed` resource to drive the installation of `KubeFed` setting the scope to be either `Namespaced` or `Cluster`. 
+    If you're planning to federate a cluster-scoped resource type like `StorageClass` then set the scope field to `Cluster`
```

Comment:
s/then/, / (Also, end sentences like this one in a period)

Reply:
Done!